CODINGSTYLE.md

```diff
@@ -254,6 +254,38 @@ int Func();
 
 * If you are writing one or more template class in the dedicated header file, use file.hpp for its name instead of file.h. This will let others know that it is template library (includes also implementation), not just header with declarations.
 
+### Code Comment Vertical Alignment
+
+When it's possible follow the existing style. If it's not possible to align any new comments with the older lines then just add a single space to the new line's comment.
```

Comment:
```suggestion
OpenTTD used to vertically-align inline Doxygen comments (example below). We have since stopped strictly following this rule to keep diffs smaller.

When you are adding code to an existing formatted section, follow the existing style if possible, without changing the formatting of other lines that are otherwise untouched. 

If your addition cannot be aligned with existing code, do not align the comments with anything and use only a single space between the code and the comment.
```

Reply:
Some of your language seemed clunky to me.  I also thought the original was a little clunky but couldn't think of anything better.

I like the latest version the best.